src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectIT.java

```diff
@@ -59,12 +58,12 @@
     @Container
     public static DockerComposeContainer<? extends DockerComposeContainer<?>> hiveContainer = new DockerComposeContainer<>(
             new File(HIVE_DOCKER_COMPOSE_YAML)) //
-                    .withExposedService(HIVE_SERVICE_NAME, HIVE_EXPOSED_PORT,
-                            Wait.forListeningPort().withStartupTimeout(Duration.ofSeconds(50)));
+            .withExposedService(HIVE_SERVICE_NAME, HIVE_EXPOSED_PORT,
```

Comment:
Indentation looks non-standard here.

Reply:
I forgot to update EclipseCodeFormatter after the IDE update and it didn't work. Fixed now